tox.ini

```diff
@@ -88,6 +91,7 @@ deps =
   sqlalchemy
   Sphinx>=1.4.4,<1.5
   sphinx_rtd_theme
+  avro
```

Comment:
As far as I can tell we're adding a whole new dependency on avro for _everybody_ installing luigi (perhaps without the slightest bit of intent to use Avro). You would need to add it in setup.py too. But it is intrusive.

A better resolution would be to have the luigi.contrib.bigquery{_avro} modules in separate packages, but to be consistent with the currently consistently used approach, I suggest we import `avro` behind a `try`, same way as `oauth2client` and `googleapiclient`. Then I think in tox.ini only gcloud environment would need the dependency.